src/bluefin_v2_client/client.py

```diff
@@ -578,7 +581,7 @@ async def get_native_chain_token_balance(self) -> float:
             result = rpc_call_sui_function(
                 self.url, callArgs, method="suix_getBalance"
             )["totalBalance"]
-            return self._from_sui_base(result)
+            return from1e18(result)
```

Comment:
this is getting native chain balance. this cannot be 1e18, it is 1e9

Reply:
Fixed